test/common/test_imccoreutils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2015 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nose.tools import *
+
+import imcsdk.imccoreutils as cutil
+
+
+def setup_func():
```

Comment:
@prvashis The `setup_func` and `teardown_func` are not required here. you should also skip passing `@with_setup` to the testcases below